.yamllint.yml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
You don't need to define a config file per repo. We have a single yamllint config defined here: https://github.com/bitrise-steplib/steps-check/blob/master/.yamllint.yml

Actually, you don't need to run `yamllint .` manually, it's part of the internal `check` step, so you can simply invoke it with `bitrise run check` (the step will apply the yamllint config from the internal step repo).

Reply:
Ah gotcha. I was running yamllint directly just because it's faster than doing the whole steps-check step, but I didn't realize there was a custom config already defined. I'll remove this.

- [ ] Remove yamllint config